README.md

```diff
@@ -83,15 +83,15 @@ curl -sSL https://install.astronomer.io | sudo bash -s -- v0.7.5
 
 > Note: Make sure you have Windows 10 and Docker installed
 
-1. Download latest release of astro-cli using this [astro_0.13.1_windows_386.zip](https://github.com/astronomer/astro-cli/releases/download/v0.13.1/astro_0.13.1_windows_386.zip)
-2. Extract `astro_0.13.1_windows_386.zip` and copy `astro.exe` somewhere in your `%PATH%`
-3. Open cmd or PowerShell console and run:
+1. Download the latest release of the Astro CLI from [this page](https://github.com/astronomer/astro-cli/releases/)
+2. Extract the file ending in `windows_386.zip` and copy `astro.exe` somewhere in your `%PATH%`
+3. Open cmd or PowerShell console and run the `astro version` command. Your output should look something like this:
 
-```
-C:\Windows\system32>astro version
-Astro CLI Version: 0.13.1
-Git Commit: 829e4702ca36dd725f1a98d82b6fdf889e5f4dc3
-```
+    ```
+    C:\Windows\system32>astro version
+    Astro CLI Version: 0.13.1
```

Comment:
Can we make this generic like X.Y.Z here or if that is too generic, refer to the latest stable release (0.23.2)

Reply:
Ooh yeah I was trying to figure out how to write a generic version without people expecting that to be the actual output, and x.y.z seems perfect for that!